cmake/GinkgoConfig.cmake.in

```diff
@@ -165,6 +165,7 @@ endif()
 if((NOT GINKGO_BUILD_SHARED_LIBS) AND GINKGO_BUILD_HIP)
     find_package(HIP REQUIRED)
     find_package(hipblas REQUIRED)
+    find_package(hipfft REQUIRED)
```

Comment:
as far as I know, they need to installed separately (at least that's what we are doing for our docker containers: https://gitlab.com/ginkgo-project/ginkgo-containers/-/blob/master/containers/gko-amd-gnu8-llvm7.baseimage

Reply:
So that would mean the hipFFT library is a new hard dependency if ginkgo is build with hip. To be honest, I'm a bit skeptical about that, since the FFT operator is a bit a-typical for ginkgo's use cases.

Reply:
Terry and I talked about that a bit - adding another dependency also makes the Spack build scripts more complex, so it might be best to add this as an optional dependency, and throw NotImplemented otherwise.

Reply:
Yes, that sounds like a good choice.